### PR TITLE
docs(feedback): re-sending of header on resume

### PR DESCRIPTION
**Pull request by dcsiebe**
Note that it is required to re-send the audio header on resume as it is
quite important.

### CODE REVIEW

**Review by joggienl — CHANGES_REQUESTED** [review]
I think #74 is created upon this one. I made some comments in there about this piece, so I leave it as "request changes" until the discussion in #73 is resolved.

**Comment by dcsiebe** [comment]
To my understanding the related discussion in #74 doesn't request changes, so what changes are requested here?

**Comment by joggienl** [comment]
That was the point of starting discussion in the other PR.

File: docs/websocket/feedback.md [hunk 1]
```diff
@@ -123,6 +127,16 @@ id   | `string` | **Required** The id of the feedback to pause.
...
+Currently **it is required to re-send the audio header when resuming
+feedback**. Failing to do so will fail the feedback as the server can't
+recognise the audio format without a header.
+
+!!! note
+    It appears to be valid to have a wave file with multiple headers.
+
+!!! warning
+    It is required to re-send the audio header upon resume. The audio
+    processing will fail if the header isn't re-sent.
```
Comment (joggienl) [inline docs/websocket/feedback.md]: Same comment as https://github.com/itslanguage/itslanguage-docs/pull/74#discussion_r181974515
It's not needed to have this redundant comment. It's already in bold above. Please remove one of the two.

**Review by joggienl — CHANGES_REQUESTED** [review]

File: docs/websocket/feedback.md (hunk 1 above)
Comment (dcsiebe) [reply docs/websocket/feedback.md]: Done

**Pull request merged**